new-module/fees.md

```diff
@@ -0,0 +1,49 @@
+---
+description: Fees and revenue in Angle _New_ Module
+---
+
+# Fees and revenue
+
+At the top level, two types of fees can be charged to users borrowing agEUR: 
+- a minting fee
```

Comment:
We should say that these are optional

Reply:
Let's talk about bad debt how it can accrue and the fact that it is well tracked by the new module